mslib/msui/topview.py

```diff
@@ -211,6 +211,9 @@ def __init__(self, parent=None, mainwindow=None, model=None, _id=None,
         # Dock windows [WMS, Satellite, Trajectories, Remote Sensing, KML Overlay, Multiple Flightpath]:
         self.docks = [None, None, None, None, None, None]
 
+        # Corresponding widget objects
+        self.widgets = [None, None, None, None, None, None]
+
```

Comment:
If I understand this right there is always a maximum of (in this case) 6 different docks/widgets that can either not exist (None) or exist (some object). Each index corresponds to a specific type of dock/widget. Is that correct?

If yes, why aren't these just 6 different named variables instead?

Reply:
This static implementation should become removed when these widgets gets plugable. At least I want to have the option that we not have to add any docking widget with all its dependencies to the core code.
To a list we could append, maybe the docks and the widgets can become a tuple or a dict.